src/ukify/ukify.py

```diff
@@ -449,7 +449,26 @@ class PeError(Exception):
 
 def pe_add_sections(uki: UKI, output: str):
     pe = pefile.PE(uki.executable, fast_load=True)
-    assert pe.FILE_HEADER.PointerToSymbolTable != 0 or len(pe.__data__) % pe.OPTIONAL_HEADER.FileAlignment == 0
+
+    # Old stubs might have been stripped, leading to unaligned raw data values, so let's fix them up here.
+    for i in range(len(pe.sections)):
+        oldp = pe.sections[i].PointerToRawData
+        oldsz = pe.sections[i].SizeOfRawData
+        pe.sections[i].PointerToRawData = round_up(pe.sections[i].PointerToRawData, pe.OPTIONAL_HEADER.FileAlignment)
+        pe.sections[i].SizeOfRawData = round_up(pe.sections[i].SizeOfRawData, pe.OPTIONAL_HEADER.FileAlignment)
+        padp = pe.sections[i].PointerToRawData - oldp
+        padsz = pe.sections[i].SizeOfRawData - oldsz
+
+        for j in range(i + 1, len(pe.sections)):
+            pe.sections[j].PointerToRawData += padp + padsz
```

Comment:
Would be simpler to keep a shift accumulator that you can just add above (and allows simplifying the loop to `for section in pe.sections`).

Reply:
Keeping an extra variable around is annoying, so I kept it this way, but simplified the overall code

Reply:
Well, you approach is more computationally expensive.